outputs.tf

```diff
@@ -2,3 +2,33 @@ output "sg_id" {
   description = "ID of the Neo4j security group"
   value       = "${aws_security_group.sg.id}"
 }
+
+output "instance_ids" {
+  description = "List of IDs of the EC2 instances"
```

Comment:
I'd make this also consistent with the rest of the List:

Reply:
done